menas/src/main/scala/za/co/absa/enceladus/menas/services/DatasetService.scala

```diff
@@ -155,12 +155,12 @@ class DatasetService @Autowired()(datasetMongoRepository: DatasetMongoRepository
         propDef.essentiality match {
           case Mandatory(true) if forRun =>
             log.warn(s"Property '${propDef.name}' is required to be present, but was not found! (Run enabled)")
-            acc
+            acc.withWarning(propDef.name, s"Property '${propDef.name}' is recommended to be present, but was not found!")
```

Comment:
the `acc.withWarning` seems correct (turning required property issue into a warning when allowing to run), but the message seems a bit off (it is actually _required_ to be present) and the fact that it is allowed to run nonetheless should be probably worded slightly differently?

Reply:
Added a hopefully better wording.